Xamarin.PropertyEditing/Properties/Resources.resx

```diff
@@ -577,4 +578,172 @@
+  </data>
+  <data name="AccessibilityActionButton" xml:space="preserve">
+    <value>Warnings</value>
+  </data>
+  <data name="AccessibilityActionButtonDescription" xml:space="preserve">
+    <value>Display warnings.</value>>
+  </data>
+  <data name="CollectionTargetsHeader" xml:space="preserve">
+    <value>Collection Items</value>
+  </data>
+  <data name="CollectionEditButton" xml:space="preserve">
+    <value>Edit</value>
+  </data>
+  <data name="ColumnResourceName" xml:space="preserve">
+    <value>Name</value>>
+  </data>
+  <data name="ColumnResourcePreview" xml:space="preserve">
+    <value>Preview</value>>
+  </data>
+  <data name="ResourceTableImageColumnTitle" xml:space="preserve">
+    <value>Icon</value>
+  </data>
+  <data name="ResourceTableTypeColumnTitle" xml:space="preserve">
+    <value>Type</value>
+  </data>
+  <data name="ResourceTableNameColumnTitle" xml:space="preserve">
+    <value>Name</value>
+  </data>
+  <data name="ResourceTableValueColumnTitle" xml:space="preserve">
+    <value>Value</value>>
+  </data>
+  <data name="CommonBrushTitleUnknown" xml:space="preserve">
+    <value>Unknown</value>
+  </data>
+  <data name="CommonBrushTitleGradient" xml:space="preserve">
+    <value>Gradient</value>>
+  </data>
+  <data name="AccessibilityObjectName" xml:space="preserve">
+    <value>{0} Object Name Editor</value>
+    <comment>Editor for Object Name Value</comment>
+  </data>
+  <data name="ObjectNamePlaceholder" xml:space="preserve">
+    <value>No Name</value>>
+  </data>
+  <data name="AccessibilityLeftEditor" xml:space="preserve">
+    <value>{0} Left Editor</value>
+    <comment>Editor for Left Value</comment>
+  </data>
+  <data name="AccessibilityTopEditor" xml:space="preserve">
+    <value>{0} Top Editor</value>
+    <comment>Editor for Top Value</comment>
+  </data>
+  <data name="AccessibilityRightEditor" xml:space="preserve">
+    <value>{0} Right Editor</value>
+    <comment>Editor for Right Value</comment>
+  </data>
+  <data name="AccessibilityBottomEditor" xml:space="preserve">
+    <value>{0} Bottom Editor</value>
+    <comment>Editor for Bottom Value</comment>
+  </data>
+  <data name="AccessibilityInputModeEditor" xml:space="preserve">
+    <value>{0} InputMode Editor</value>
+    <comment>Editor for InputMode Value</comment>
+  </data>
+  <data name="AccessibilityPathEditor" xml:space="preserve">
+    <value>{0} {1} Path Editor</value>
+    <comment>Editor for File/Directory Path Editor</comment>
+  </data>  
+  <data name="ChooseFileOrDirectory" xml:space="preserve">
+    <value>Choose a {0}</value>
+    <comment>Choose a File or Directory</comment>
+  </data>
+  <data name="BrowseFileOrDirectory" xml:space="preserve">
+    <value>Browse for a {0}</value>
+    <comment>Browse for a File or Directory</comment>
+  </data>
+  <data name="RevealFileOrDirectory" xml:space="preserve">
+    <value>Reveal {0}</value>
+    <comment>Reveal a File or Directory</comment>
```

Comment:
I'm just reviewing the strings now and i see a problem - we can't use string concatenation to produce localized values.

If we need the strings `Reveal a File` and also `Reveal a Directory` then to properly translate those we need to have two different strings in our resx and then choose the appropriate one in code. What we *cannot* do is the equivalent of:

```
myLabel.Text = string.Format (Properties.Resources. RevealFileOrDirectory, IsDirectory ? Properties.Resources.Directory : Properties.Resources.File)
```
This will not produce a valid string in languages other than english. We should review in a separate PR to ensure we use whole strings instead of concatenation :)